src/importkeydialog.cpp

```
@@ -0,0 +1,141 @@
+ImportKeyDialog::~ImportKeyDialog() = default;
+
+QString ImportKeyDialog::importedKeyFingerprint() const {
+  return m_importedFingerprint;
+}
+
+void ImportKeyDialog::on_fileButton_clicked() {
+  const QString fileName = QFileDialog::getOpenFileName(
+      this, tr("Import GPG Key"), QString(),
+      tr("GPG Key Files") + " (*.asc *.gpg *.key);;;" + tr("All Files") + " (*)");
+
+  if (fileName.isEmpty()) {
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)